src/app/containers/InlineLink/index.jsx

```diff
@@ -32,13 +32,23 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
     );
   }
 
-  // else return a normal hyperlink
+  if (isExternal) {
```

Comment:
I don't think you need this condition. You could have the single return block as before, and just do

```js
<InlineLink href={locator} aria-label={isExternal ? 'foo' : null}>
```

Reply:
To make this clearer to read, defining the variable above would be better & using deepGet for the text:
```
const linkText = deepGet([0, 'model', 'text'], blocks);
const ariaLabel= isExternal ? `${linkText}${externalLinkText}`: null;
```